### PR TITLE
fixed roadmap responsiveness on mobile

### LINKED ISSUE #2183: Roadmap mobile version is not awesome
**Vuestic-ui version:** 1.4.8

image

### PR DESCRIPTION
**Pull request by ameer-khan-ashraf**
Close #2183 

## Description
Added two media query in the roadmap.vue file to fix the roadmap in mobile view

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/page-configs/introduction/roadmap/components/roadmap.vue [hunk 1]
```diff
@@ -82,10 +82,18 @@ $gap: 1rem;
...
+    }
+
     .docs-roadmap__column {
       &-wrapper {
         width: 33%;
         padding: calc($gap / 2);
+
+        @media screen and (max-width: 479px) {
+          width: 100%;
+        }
```
Comment (m0ksem) [inline packages/docs/src/page-configs/introduction/roadmap/components/roadmap.vue:96]:
```suggestion
    .docs-roadmap__column {
      &-wrapper {
        width: 100%;
        min-width: 300px;
        flex: 1;
        padding: calc($gap / 2);
```

File: packages/docs/src/page-configs/introduction/roadmap/components/roadmap.vue [hunk 2]
```diff
@@ -82,10 +82,18 @@ $gap: 1rem;
     display: flex;
     width: 100%;
 
+    @media screen and (max-width: 479px) {
+      flex-direction: column;
+    }
```
Comment (m0ksem) [inline packages/docs/src/page-configs/introduction/roadmap/components/roadmap.vue:87]:
```suggestion
    display: flex;
    flex-wrap: wrap;
    width: 100%;
```

**Review by m0ksem — CHANGES_REQUESTED** [review]
Thanks!

I would suggest you make these columns responsive to any screen size, instead of anchoring to a specific screen size.

Having flex-wrap will move column that not fit downside, flex 1 will force column to take as much space as it can, and min width will force columns to be readable and move other columns to down.

**Comment by ameer-khan-ashraf** [comment]
Thanks for the heads up. I have updated the code and pushed another commit.

**Review by m0ksem — APPROVED** [review]

**Pull request merged**